### DISCUSSION TITLE
changes not reflecting after build with apktool

### DISCUSSION THREAD

**Discussion by thekishan** [iBotPeaches/Apktool#3302] [Q&A] [answered]
I decompiled my old apk with dex2jar because i needed java code. and made some changes in code. And built new apk with apktool (2.8.1). Before sometime, it was working fine res and assets folders but now im making changes in java files but they are not reflecting in build. When i check again by decompiling new apk, its not showing. Only showing the old code in file. Any solution?

**Comment by iBotPeaches** [comment, marked as answer]
I'm not following your explanation, but it doesn't seem like it would work. Apktool disassembles to `.smali` files and re-assembles those. If you provided the output of dex2jar and attempted to just "build" that with Apktool - it would never work.

So closing as this appears to be unsupported behavior.
